.github/jobs/macos_cmake.yml

```
@@ -0,0 +1,15 @@
+parameters:
+  vmImage: ''
+
+steps:
+  - script: |
+      wget https://github.com/Kitware/CMake/releases/download/v$(CMAKE_VERSION)/cmake-$(CMAKE_VERSION)-macos-universal.tar.gz
+      sudo mkdir -p /opt/cmake
```

do we want to remove /opt/cmake if it already exists?
```suggestion
      sudo rm -rf /opt/cmake
      sudo mkdir -p /opt/cmake
```

---

/opt should be empty by default in the VMs

---

Ok, I wasn't sure what the lifecycle for a VM was, if it was recreated from an image for each pipeline run, or only periodically